votes/0639-propose-a-release-note-guide.md

```diff
@@ -0,0 +1,76 @@
+
+## Bug fix
+
+A bug fix note means that your PR fixes an existing bug or issue. This type of notes start with "Fix" followed by "the issue/bug".
+
+Write your note clearly and adequately so that your target readers can get the main point of your bug fix. The bug or issue must be directly perceivable to the users, and you can refer to the associated GitHub issues.
+
+**Examples:**
+
+| **Not recommended**                                          | **Clear in type** | **Adequate and clear in meaning** | **User perspective** | Recommended                                                  |
+| ------------------------------------------------------------ | ----------------- | --------------------------------- | -------------------- | ------------------------------------------------------------ |
+| lock_resolver: avoid pessimistic transactions using resolveLocksForWrite | ❌                 | ✅                                 | ❌                    | Fix the issue that committing pessimistic transactions might cause write conflict |
+| retry when meeting stablish conn fails                       | ❌                 | ❌                                 | ❌                    | Fix the issue of unexpected results when TiFlash fails to establish MPP connections |
+| Fix the issue that greatest(datetime) union null returns empty string | ✅                 | ❌                                 | ✅                    | Fix the issue that the query result might be wrong when NULL is in the UNION subquery |
+| copr: make CM Sketch built with the same encoding as what TiDB assumes | ❌                 | ❌                                 | ❌                    | Fix the wrong statistics caused by the format inconsistency of CM Sketch between TiKV and TiDB |
+
+## Improvement
+
+An improvement note means that your PR improves stability or performance of the product, or enhances an existing feature. In addition to describing what your PR has changed, you should also mention how users can benefit from it.
+
+This type of release note consists of two parts: what you have changed + the benefit of your change. This type of release notes often starts with "support", "increase", "improve", "optimize", etc.
+
+**Examples:**
+
+| **Not recommended**                                          | **Clear in type** | **Adequate and clear in meaning** | **User perspective** | Recommended                                                  |
+| ------------------------------------------------------------ | ----------------- | --------------------------------- | -------------------- | ------------------------------------------------------------ |
+| Not use the stale read request's `start_ts` to update `max_ts` to avoid commit request keep retrying | ✅                 | ✅                                 | ✅                    | Avoid excessive commit request retrying by not using the Stale Read request's start_ts to update `max_ts` |
```

Comment:
```suggestion
| Not use the stale read request's `start_ts` to update `max_ts` to avoid commit request keep retrying | ✅                 | ✅                                 | ❌                    | Improve commit performance in some edge cases |
```

Reply:
@NingLin-P @sticnarf PTAL about whether this change is correct.

Reply:
lgtm

Reply:
In fact, I think it's a bug fix. If `max_ts` is updated from the `start_ts` of a stale read request, a later reader may not see committed data.

See https://github.com/pingcap/tidb/issues/25680